### DISCUSSION TITLE
InjectingScript using contentScript

### DISCUSSION THREAD

**Discussion by NorbertBodziony** [crxjs/chrome-extension-tools#236] [Q&A]
I am looking for a way to inject script into running web page. 
I used to do it like this with webpack but here idk how to bundle inject.js so content will have access to it.
```function injectScript(filePath: string): void {
  var script = document.createElement('script')
  script.setAttribute('type', 'text/javascript')
  script.setAttribute('src', filePath)
  document.documentElement.appendChild(script)
}
injectScript(browser.runtime.getURL('inject.js'))
```
Here is repo that i have tried https://github.com/NorbertBodziony/vite-extension/tree/inject

**Comment by jacksteamdev** [comment]
This hasn't been implemented yet, but it's coming in the near future! I plan to work on it next week; details depend on finalizing #224 

The plan is to support injected scripts by adding the script to `manifest.web_accessible_resources`, then doing things like you mentioned above.

**Comment by NorbertBodziony** [comment]
Great i appreciate

**Comment by gorbit99** [comment]
Any update on this since? I'd love to use this plugin with vite, but currently adding the script to Web accessible resources won't make it export. Should I go back to using webpack for the time being?
